beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -168,13 +180,104 @@ def normalize(tensor):
 print("Median relative difference between original and MuLaw reconstucted signals: {:.2%}".format(err))
 
 
+######################################################################
+# Functional
+# ---------------
+# 
+# ``torchaudio`` supports a growing list of
+# `functions <https://pytorch.org/audio/functional.html>`_ to perform common 
+# audio operations. 
+# 
+# -  **istft**: Inverse short time Fourier Transform.
+# -  **spectrogram**: Create a spectrogram from a raw audio signal.
+# -  **amplitude_to_DB**: Turns a tensor from the power/amplitude scale to the
+#    decibel scale.
+# -  **create_fb_matrix**: Create a frequency bin conversion matrix.
+# -  **create_dct**: Creates a DCT transformation matrix with shape ``(n_mels, n_mfcc)``,
+#    normalized depending on norm.
+# -  **mu_law_encoding**: Encode signal based on mu-law companding.
+# -  **mu_law_decoding**: Decode mu-law encoded signal.
+# -  **complex_norm**: Compute the norm of complex tensor input.
+# -  **angle**: Compute the angle of complex tensor input.
+# -  **magphase**: Separate a complex-valued spectrogram with shape (*, 2) into its
+#    magnitude and phase.
+# -  **phase_vocoder**: Given a STFT tensor, speed up in time without modifying pitch by
+#    a factor of ``rate``.
+# -  **gain**: Applies amplification or attenuation to the whole waveform.
+# -  **dither**: Increases the perceived dynamic range of audio stored at a
+#    particular bit-depth.
+# -  **compute_deltas**: Compute delta coefficients of a tensor.
+# -  **equalizer_biquad**: Design biquad peaking equalizer filter and perform filtering.
+# -  **lowpass_biquad**: Design biquad lowpass filter and perform filtering.
+# -  **highpass_biquad**:Design biquad highpass filter and perform filtering.
+#
+# You may notice that there are similar operations in ``torchaudio.functional`` and 
+# ``torchaudio.transforms``. To differentiate, think of ``torchaudio.functional`` as a
+# `computation` on a waveform, and ``torchaudio.transforms`` as something affecting the
+# `state` of a waveform.
+# 
+# Let's experiment with a few of the functionals and visualize their output. Taking our 
+# spectogram from the last example, we can compute it's deltas:
+
+computed = torchaudio.functional.compute_deltas(specgram, win_length=3)
+print("Shape of computed deltas: {}".format(computed.shape))
+
+plt.figure()
+p = plt.imshow(computed.log2()[0,:,:].detach().numpy(), cmap='gray')
+
+######################################################################
+# We can take the original waveform and apply different effects to it.
+#
+
+GAIN_DB = 5.0
+gain_wf = torchaudio.functional.gain(waveform, GAIN_DB)
```

Comment:
Let's avoid the abbreviation and be consistent with what is below: `gain_waveform`, `dither_waveform`, etc.

How about `gain_waveform = torchaudio.functional.gain(waveform, gain_db=5.0)`?